xml/System.Collections.Immutable/ImmutableArray`1.xml

```diff
@@ -325,9 +325,9 @@
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="TOther">To be added.</typeparam>
+        <typeparam name="TOther">The type of array element to be returned.</typeparam>
         <summary>Initializes a new instance of the <see cref="T:System.Collections.Immutable.ImmutableArray`1" /> struct by casting the underlying array to an array of type <typeparamref name="TOther" />.</summary>
-        <returns>To be added.</returns>
+        <returns>An ImmutableArray instance with elements cast to the new type.</returns>
```

Comment:
Either use a see cref, or use natural language. Usually, natural language is preferred, especially if we are already in the context where the user knows what API or type you're talking about:
```suggestion
        <returns>An immutable array instance with elements cast to the new type.</returns>
```